### PR TITLE
fix(RSS3): mf-3313 some small issues

### PR DESCRIPTION
**Pull request by UncleBill**
https://mask.atlassian.net/browse/MF-3313

### CODE REVIEW

File: packages/web3-providers/src/types/RSS3.ts [hunk 1]
```diff
@@ -189,13 +189,11 @@ export namespace RSS3BaseAPI {
...
-            {
-                /** URL */
-                address: string
-                mime_type: 'image/png' | string
-            },
-        ]
+        media: Array<{
+            /** URL */
+            address: string
+            mime_type: 'image/png' | string
```
Comment (guanbinrui) [inline packages/web3-providers/src/types/RSS3.ts:195]: Make the editor happy.

```suggestion
            mime_type: 'image/png' | Omit<string, 'image/png'>
```

Comment (UncleBill) [reply packages/web3-providers/src/types/RSS3.ts:195]: Got such error:
image

Comment (Jack-Works) [reply packages/web3-providers/src/types/RSS3.ts:195]: `'image/png' | string` is equal to `string`

Comment (guanbinrui) [reply packages/web3-providers/src/types/RSS3.ts:195]: > Got such error:
> 
> image
> 
> 

Omit will help auto completion. Maybe cast it to string?

Comment (UncleBill) [reply packages/web3-providers/src/types/RSS3.ts:195]: > `'image/png' | string` is equal to `string`

I know, just want to document it in code.

Comment (UncleBill) [reply packages/web3-providers/src/types/RSS3.ts:195]: Would like to list as many types as possible.

**Pull request merged**